routes/mobilize/cohort/index.js

```diff
@@ -41,10 +41,10 @@ module.exports = (req, res) => {
 				}).catch(err => console.log(err)))
 			} else if (pinboard) {
 				batch.push(ownDB().then(async ownId => {
-					const padlist = await DB.general.any(`
+					const padlist = (await DB.general.any(`
 						SELECT pc.pad FROM pinboard_contributions pc
 						WHERE pc.pinboard = $1 AND pc.db = $2
-					;`, [ pinboard, ownId ]).map((row) => row.pad);
+					;`, [ pinboard, ownId ])).map((row) => row.pad);
```

Comment:
missing (...) making `map` trying to work on the promise instead of *awaiting* the result